### DISCUSSION TITLE
how to spawn windows on non-primary monitor?

### DISCUSSION THREAD

**Discussion by ExecutorElassus** [fvwmorg/fvwm3#792] [Q&A]
My xorg setup treats my two physical monitors as a single virtual display (I have no config file for the display, so it does it using default settings). When fvwm(-2.7.0) would start, IconMan would be in the bottom-left corner of the (double-wide) desktop, and ButtonsBar would be in the bottom-right. Also, programs set to use full-screen display would spread across both monitors (nice for movies, not so nice for games that didn't have double-wide resolutions). 

Now, when fvwm3 starts, both IconMan and ButtonsBar are spawned on the left display. Furthermore, full-screen apps will only display on the currently active monitor (nice for single-width games, not so nice for movies). Also, windows spawned by programs (say, popup windows) that fvwm3 places itself always spawn only on the active display, which clutters up the screen.

Is there a way to configure fvwm3 so that it doesn't do this? I preferred the old behavior. 

Cheers,

**Comment by lgsobalvarro** [comment]
You could make style for those applications you want to always start in the other screen.
For example:
`Style Foo Geometry 64x256-0+0@HDMI-1`
Foo will be place at -0+0 in HDMI-1 (My exterman screen) with a Geometry of 64x265.
